### DISCUSSION TITLE
桥梁建筑师 闪退

### DISCUSSION THREAD

**Discussion by Ah-Bo** [Gamuxorg/bbs#300] [游戏或网站运行反馈]
### 以下是推荐模板
游戏名称：
桥梁建筑师
操作系统：*（uname -a）*
Linux pc 4.19.66-1-MANJARO #1 SMP PREEMPT Fri Aug 9 18:01:53 UTC 2019 x86_64 GNU/Linux
bug描述：*（具体说说发生了什么）*
闪退
终端运行输出 ：*（如果有的话）*
Set current directory to /tmp/.mount_downloSoPjX0/usr/bin
Found path: /tmp/.mount_downloSoPjX0/usr/bin/polybridge.x86_64
Mono path[0] = '/tmp/.mount_downloSoPjX0/usr/bin/polybridge_Data/Managed'
Mono path[1] = '/tmp/.mount_downloSoPjX0/usr/bin/polybridge_Data/Mono'
Mono config path = '/tmp/.mount_downloSoPjX0/usr/bin/polybridge_Data/Mono/etc'
displaymanager : xrandr version warning. 1.6
client has 2 screens
displaymanager screen (0)(LVDS-1): 1280 x 800
Using libudev for joystick management

Importing game controller configs

**Comment by hyh19962008** [comment]
log显示有两个屏幕，只留一个屏幕试试？

**Comment by Ah-Bo** [comment]
@hyh19962008 但我只有一个屏幕（物理

**Comment by hyh19962008** [comment]
试试解压后运行，加参数--appimage-extract 解压

**Comment by hyh19962008** [comment]
但我觉得可能还是屏幕或者分辨率的问题，log倒数第四行有xandr warning的警告

**Comment by Ah-Bo** [comment]
> 试试解压后运行，加参数--appimage-extract 解压

解压后依然无用，一样的log，一样的闪退

**Comment by Ah-Bo** [comment]
> 但我觉得可能还是屏幕或者分辨率的问题，log倒数第四行有xandr warning的警告

要开学了，不搞了
